docs/contributing/how-to-label-an-issue.md

```diff
@@ -28,21 +30,24 @@ You can get an invite to the team by having a Pull Request merged into the Gatsb
 
 ## How to label an issue
 
-Ideally, every issue should have a single `type:` label applied to it. Optionally a `status:` label or other labels may also be applied.
+When a new issue is filed, our Gatsbot applies a `status: triage needed` label to it. Triage is undertaken every workday by Gatsby Core Team members.
 
-Before continuing, get familiar with [Gatsby's issue labels and their descriptions](https://github.com/gatsbyjs/gatsby/issues/labels).
+Triaging new issues involves identifying each issue's type (such as feature request or bug report), topic (such as GraphQL or Gatsby Recipes), and status. These labels are then part of a broader issue management workflow undertaken by the rest of the Gatsby team.
 
-The broad steps to labeling an issue are:
+Before continuing, get familiar with [Gatsby's issue labels and their descriptions](https://github.com/gatsbyjs/gatsby/issues/labels), particularly the `type:`, `topic:`, and `status:` labels.
+
+The general steps to triaging an issue are:
 
 - Read an issue
 - Choose the labels that apply to that issue
+- Include any helpful comments that guide the author toward the next steps they need to take
 - That's it - sit back and relax, maybe take a few moments to enjoy the satisfaction of a job well done
 
 The rest of this document will describe how to choose the right labels for an issue.
 
 ### Find an issue that you're interested in
 
-Start with [Gatsby's issues list](https://github.com/gatsbyjs/gatsby/issues) and scroll through until you see a recent one that strikes your interest. Alternatively, you can view the [list of unlabelled issues](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aopen+is%3Aissue+no%3Alabel).
+Start with [Gatsby's issues list](https://github.com/gatsbyjs/gatsby/issues), specifically the [list of untriaged issues](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+triage+needed%22).
```

Comment:
The retext-spell linter is failing on the word `untriaged`, which needs to be added to the dictionary: https://github.com/gatsbyjs/gatsby/blob/master/dictionary.txt